src/GeositeFramework/plugins/launchpad/main.js

```diff
@@ -86,6 +86,33 @@ define([
                 }));
 
                 $(this.container).empty().append($el);
+
+                // add truncate listeners to each toggle botton
+                $(".truncate-container").each(function () {
+                    if (parseFloat($('p', this).css("height")) > 50) {
+                        $('p', this).each(function() {
+                            this.classList.toggle('truncate-box');
+                        });
+
+                        this.addEventListener('click', function(ev) {
+                            if (ev.target.tagName === 'BUTTON') {
+                                $('p', this).each(function() {
+                                    this.classList.toggle('truncate-box');
+                                });
+                                $('.truncate-toggle', this).each(function() {
+                                    if (this.textContent === 'View more') {
+                                        this.textContent = 'View less';
+                                    } else {
+                                        this.textContent = 'View more';
+                                    }
+                                });
+                            }
+                        });
+                    } else {
+                        $(".truncate-toggle", this).css({ display:"none" });
+                    }
```

Comment:
Couple of suggestions:

* This code does exactly what #749 suggests ("If the plugin description is greater than a given height ..."). However, I would recommend using text length instead of DOM height. Then we could move half of this logic to the template.

  Ex.

  ```
  <p><%= shortDescription %></p>
  <% if (description.length > 100) { %>
  <button class="button truncate-toggle">View more</button>
  <% } %>
  ```

* Consider moving the click events to the `bindEvents` function. We try to use live selectors whenever possible.
* For parity with the rest of the codebase, prefer to use jQuery functions to manipulate DOM attributes (ex. `$(this).text(...)` instead of `this.textContent`, and `$(this).toggle(...)` instead of `this.classList.toggle`)
* Defining hard-coded strings in code instead of templates presents some internationalization difficulties. Consider using an ellipsis instead to avoid this problem? Maybe even get rid of the option to "View Less" to keep things simple.